research-hub-web/src/app/components/home/home.component.html

```diff
@@ -23,7 +23,7 @@
         <!-- Title -->
         <div class="text site-padding" ngClass.lt-md="site-padding-mobile">
           <h2>{{ title }}</h2>
-          <p>{{ description }}</p>
+          <p>We'd love to hear from you. Get in contact via email <a href='mailto: reshubproject@auckland.ac.nz'>reshubproject@auckland.ac.nz</a>.</p>
```

Comment:
Can you move the text to home.component.ts - there's a few variables in there under 'Contact Section'. Then you can leave the description variable here as is.